script/install/csharp/dotnet-csc

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# This script does two things
+#   1. Redirects all output to stderr
+#   2. Supplies .dll references to the csc command
+
+# Actually compile:
+# [dotnet] [csc] -reference:[stdlib.dll] [...passed in args...]
```

Comment:
Does this intentionally say "stdlib.dll" instead of "netstandard.dll"?
I like "stdlib" (without .dll) because it makes more sense to me than "netstandard.dll", but with .dll it sounds like it's referring to a specific file rather than describing the purpose of a file.

Reply:
The "stdlib" is intentional but I don't think the ".dll" is. I agree with "stdlib" only.